### PR TITLE
Issue-1-create-an-entry

### PR DESCRIPTION
**Pull request by Raphael-Cruzado**
code-journal-ft1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,8 +16,39 @@
...
+      </div>
+    </div>
+
   </header>
   <main>
+    <div class="container" data-view="entry-form">
+      <form action="" method="get">
+        <h1>New Entry</h1>
+        <div class="row">
+          <img class="column-half" id="display-image" src="images/placeholder-image-square.jpg" alt="user-image">
```
Comment (BrettAlbright) [inline index.html]: the img element should not be a column, it should be inside of a column.

File: index.html [hunk 2]
```diff
@@ -16,8 +16,39 @@
...
+          <div class="column-half">
+            <div class="column-half">
```
Comment (BrettAlbright) [inline index.html]: you don't need to set a column class inside of one, you just want to have the labels and inputs in here, and set them to be display block and 100% width so they take up the full column and stack

File: js/main.js [hunk 3]
```diff
@@ -1,2 +1,27 @@
...
+    $imageDisplay.src = 'images/placeholder-image-square.jpg';
+  }
+});
+
+var $titleBox = document.querySelector('#input-box');
+var $notesBox = document.querySelector('textarea');
+var $form = document.querySelector('form');
+
+$form.addEventListener('submit', function (e) {
+  e.preventDefault();
```
Comment (BrettAlbright) [inline js/main.js:16]: one this to add to this function is to reset the placeholder image after the form is submitted so that it matches the example.

**Review by Zhiwutian — APPROVED** [review]
Looking Good! A few updates are needed

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small changes needed

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**